crates/miden-objects/Cargo.toml

```diff
@@ -48,5 +48,7 @@ anyhow = { version = "1.0.93", default-features = false, features = ["std", "bac
 assert_matches = { workspace = true }
 criterion = { version = "0.5", default-features = false, features = ["html_reports"] }
 miden-objects = { path = ".", features = ["testing"] }
+miden-air ={ version = "0.12" }
 rstest = { version = "0.23" }
 tempfile = { version = "3.14" }
+winter-air = { version = "0.11" }
```

Comment:
Why do we need these new dependencies?

Reply:
I ended up removing `miden-air` from here because I was able to do `Default::default` for the `HashFunction`, though I'm using `winter-air` for the `Proof`.